main/EventSystem.cpp

```diff
@@ -370,6 +384,51 @@ std::string CEventSystem::LowerCase(std::string sResult)
 	return sResult;
 }
 
+// Temporarily function used for removing leftover dzVents files
+int CEventSystem::RemoveDir(const std::string &dirnames, std::string &errorPath)
```

Comment:
Could you move this function to Helper.cpp, we might need this later for other purposes and it's better to have this global